mysite/models.py

```diff
@@ -1,47 +1,80 @@
 from django.db import models
+from django.urls import reverse
+import datetime
+from django.utils import timezone
+import pytz
+utc=pytz.UTC
 
-
-class Image(models.Model):
-    img = models.ImageField(upload_to='images/gallery')
-    alt_text = models.CharField(max_length=50)
-    display_on_index = models.BooleanField()
-
+class Member(models.Model):
+    roll_number = models.CharField(max_length=8, primary_key=True)
+    member_name = models.CharField(max_length=25)
+    post = models.CharField(max_length=300, blank=True, help_text='Insert past post names also. ex, <batch> Convener for a past batch convener')
+    insta = models.URLField(default=" ")
+    email = models.EmailField(max_length = 50,default=" ")
+    batch = models.CharField(verbose_name='batch', max_length=4, help_text="passing year")
+    head = models.BooleanField(default=False)
+    testimonial = models.TextField(default=" ")
+    member_img = models.ImageField(upload_to='images/members')
+    active = models.BooleanField(verbose_name="member status", default=True)
     def __str__(self):
-        return self.alt_text
-
+        return self.member_name
+    @property
+    def alumni_filter(self):
+        curr_year = datetime.datetime.now().year
+        return self.batch + 4 < curr_year and self.batch < curr_year
 
 class Event(models.Model):
-    page_to_display = models.IntegerField(default=0)
-    bg_img = models.ImageField(upload_to='images/event')
-    small_text = models.CharField(max_length=50)
-    created_on = models.DateTimeField(auto_now_add=True)
+
+    title = models.CharField(max_length=50)
+    description = models.TextField(default=" ")
+    date = models.DateTimeField(auto_now_add=False, default=timezone.now())
+    resources = models.CharField(verbose_name='Event Resources', max_length=100, blank=True)
+    @property
+    def is_past(self):
+        utc_date = utc.localize(datetime.datetime.now())
+        return utc_date < self.date
 
     def __str__(self):
-        return self.small_text
+        return self.title
 
 
-class Member(models.Model):
-    roll_number = models.CharField(max_length=8, primary_key=True)
-    member_name = models.CharField(max_length=50)
-    post = models.CharField(max_length=300)
-    insta = models.URLField(default=" ")
-    linkdin = models.URLField(default=" ")
-    head = models.BooleanField(default=False)
-    testimonial = models.TextField(default=" ")
-    member_img = models.ImageField(upload_to='images/members')
+class Art(models.Model):
+    SKT = 'Sketches'
+    DDL = 'Doodles'
+    ABS = 'Abstract'
+    PRT = 'Portrait'
+    NTR = 'Nature'
+    DGT = 'Digital'
+    OTR = 'Other'
+    art_choices = [
+        (SKT, 'Sketches'),
```

Comment:
@PiyushPI07 Do you this this would be a better way?
Same goes for all the models with such choices.
```
CHOICES = [ (1,'Sketches'),(2,'Doodles'),(3,'Abstract'),(4,'Portrait'),(5,'Nature'),(6,'Digital'),(7,'Other')
art_type = models.CharField(verbose_name='Type', max_length=20, choices=CHOICES)
```